### ISSUE TITLE
Unable to have a top-level ordered list in a List block

### ISSUE THREAD

**Issue by yandodov** [getkirby/kirby#3198]
**Describe the bug**  
Whatever I type in a List block creates an unordered list, even if I type `1.` followed by space. I can only have an ordered list when it's nested under an unordered list item.

**To Reproduce**  
1. Open a Blocks field in the panel
2. Attempt to write an ordered list

**Expected behavior**  
You should be able to create either an ordered or unordered list.

**Kirby Version**  
3.5.3.1

**Comment by afbora** [comment]
@hdodov Am i missing something?

![image (3)](https://user-images.githubusercontent.com/3393422/110661137-504a5d00-81d5-11eb-8081-bb1fddb97124.png)

**Comment by lukasbestle** [comment]
I agree with @hdodov that the current behavior is not really intuitive. You need to create a list block, then type something, select the text and finally change the list type from the toolbar.

For example in Notion there are way fewer steps: If you type `-`, you get an unordered list and if you type `1. `, you get an ordered list. Without using the mouse at all.

**Comment by yandodov** [comment]
I had no idea you get an options bar when you select the text, to be honest. I don't think it's good to put such an important functionality in a place that people might not know it even exists.

**Comment by bastianallgeier** [comment]
I agree that this isn't solved perfectly yet. We need to work on this.

**Comment by christophlocher** [comment]
Just want to put this back on the map – I really struggled with this too until I found this issue by googling. I always looked for it in the block settings (as in the overlay that opens up when you click the edit icon of a block), because it feels like a block-level option.

Also when I press CMD+A to select all and then try to change it with the toolbar it does not work, only when I use the mouse to select the text. (In Chrome)

**Comment by bvdputte** [comment]
Same as @ChristophLabacher 

Can you please reopen this?

**Comment by distantnative** [comment]
🤫


https://user-images.githubusercontent.com/3788865/228045054-24d099bc-bcb3-4995-9e4b-12f7cdc11b04.mov

**Comment by gaufde** [comment]
I'll just throw my hat in the ring by saying I couldn't figure out how to change the list type until I found this issue as well.

**Comment by afbora** [comment]
I think this could be solved more simply by not forcing the List block to start as either `ul` or `ol`. Instead, it could start in a neutral paragraph state and only be promoted once the user’s intent is clear: if the input begins with `1. ` or `1- `, convert it to an ordered list; if the user starts typing regular text or a bullet marker, convert it to an unordered list. That would avoid the current top-level ordered-list issue without adding much extra complexity, and it would feel more natural when typing from scratch.

**Comment by distantnative** [comment]
@afbora But then the list field is not ensured anymore to only have lists as content. But could also be saved with just a `<p>Text</p>`

**Comment by afbora** [comment]
@distantnative That’s true and I agree that the field still needs to guarantee list-only content at save time. My suggestion was only about the editing state: start in a neutral paragraph internally so the first keystrokes can disambiguate between ordered and unordered lists more naturally, but prevent persisting plain `<p>` content by normalizing or rejecting it before the value is saved. So the saved output would still remain list-only.
